apps/comps/e2e/utils/rpc-client-helpers.ts

```diff
@@ -48,6 +69,63 @@ function createMockCookies(privyToken?: string) {
   };
 }
 
+/**
+ * Create a mock headers object for testing
+ */
+function createMockHeaders(customHeaders?: Record<string, string>): Headers {
```

Comment:
can we use our test framework's built-in mocking functionality for this?

Reply:
Good point! I'll use `new Header` instead.